app/bundles/CoreBundle/Assets/js/1.core.js

```diff
@@ -581,8 +581,6 @@ var Mautic = {
                     textarea.on('froalaEditor.focus', function (e, editor) {
                         Mautic.showChangeThemeWarning = true;
                     });
-
-                    textarea.froalaEditor(mQuery.extend(Mautic.basicFroalaOptions, options));
```

Comment:
Just verifying here that this wasn't removed on accident as part of a merge - I think staging still has it https://github.com/mautic/mautic/blob/staging/app/bundles/CoreBundle/Assets/js/1.core.js#L585. If done purposively, ignore this  :-)